lib/token.js

```diff
@@ -464,53 +511,81 @@ function getWithoutPrompt(sdk, oauthOptions, options) {
 function getWithPopup(sdk, oauthOptions, options) {
   var oauthParams = util.clone(oauthOptions) || {};
   util.extend(oauthParams, {
-    display: 'popup'
+    display: 'popup',
+    responseMode: 'okta_post_message'
```

Comment:
why you have to add `responseMode: 'okta_post_message'`?

Reply:
In an earlier version of this PR, I was changing the default responseMode, so this line was necessary.  That breaking change has been removed.  Also I have discovered the true reason `okta_post_message` is the default, is because token.renew() cannot take any parameters, so must use defaults or values set in the constructor.